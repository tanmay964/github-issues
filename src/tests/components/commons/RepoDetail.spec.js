import React from 'react';
import renderer from 'react-test-renderer';
import RepoDetail from '../../../components/commons/RepoDetail';

it('renders correctly', () => {
  const tree = renderer
    .create(<RepoDetail tag="Test" value="snapshot" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
