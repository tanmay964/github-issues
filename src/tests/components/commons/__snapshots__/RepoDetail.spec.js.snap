// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-ifAKCX EpfhZ"
>
  <span
    className="sc-EHOje hYVzSu"
  >
    Test
    <span
      className="sc-bdVaJa csInxQ"
    />
  </span>
  <span
    className="sc-bZQynM caDpFp"
  >
    snapshot
  </span>
</div>
`;
